### ISSUE TITLE
cub200-2011，config文件中的参数设置没变batch=16，双卡训练，准确率最高在0.869。

### ISSUE THREAD

**Issue by weihua93** [yangze0930/NTS-Net#20]
batchsize=16,两块GPU训练，学习率0.001，达不到论文中的0.875的准确率。请问，你们达到的是怎么设置的参数？训练了多少epoch?

**Comment by yaxingwang** [comment]
Hi @weihua93 
How many epochs you conduct when the accuracy is 0.869?
Thanks

**Comment by weihua93** [comment]
> Hi @weihua93
> How many epochs you conduct when the accuracy is 0.869?
> Thanks

183

**Comment by yaxingwang** [comment]
Thanks

**Comment by woshiyanyan** [comment]
hi, I have some problems on the pretrained model of ResNet50 on ImageNet, which pretrained model did you use, the 224*224 one or the 448*448 one?

**Comment by yaxingwang** [comment]
I used 448

**Comment by woshiyanyan** [comment]
> I used 448

Thank you!  While I can't find the pretrained model with size of 448, Can you send me one? my email is hywu@mail.bnu.edu.cn

**Comment by yaxingwang** [comment]
The provided one is 448

**Comment by woshiyanyan** [comment]
> The provided one is 448
![image](https://user-images.githubusercontent.com/32740101/61588227-386e3180-abca-11e9-8b83-858e47ba4a5d.png)
you mean here provided the one of 448, I just can't open the link

**Comment by yaxingwang** [comment]
I have sent your email. Please check it.

Let me know your result from scratch if you train model

**Comment by woshiyanyan** [comment]
> I have sent your email. Please check it.
> 
> Let me know your result from scratch if you train model
Sorry，I still can't open the link you send me.
![image](https://user-images.githubusercontent.com/32740101/61588607-33f94700-abd1-11e9-9772-61eda9435377.png)
So sorry to touble you.

**Comment by GuohongWu** [comment]
+1, 试了多种batch_size，多卡组合，准确率最高只能到0.868

**Comment by JingyunLiang** [comment]
I got the best accuracy of 87.3% (K=4) in epoch 127 following default setting. Although the accuracy fluctuates between 86.8% and 87.3% from epoch 127 to epoch 156 (I stopped in epoch 156).

**Comment by RYYAI** [comment]
@MichaelLiang12 Did you use the defaulted parameters to get  87.3%? or made some change on the config ?

**Comment by JingyunLiang** [comment]
Just as I said, I followed default setting with no changes.
